lib/Server.js

```diff
@@ -35,7 +35,16 @@ class Server {
     // Keep track of websocket proxies for external websocket upgrade.
     this.webSocketProxies = [];
     this.sockets = [];
+
+    // TODO: remove this after plugin support is published
+    if (compiler) {
```

Comment:
I think we should keep it here to avoid breaking change

Reply:
Sure, will remove comment.

Reply:
Done.